clients/scheduler/src/components/time-slot-select/index.tsx

```diff
@@ -72,5 +75,28 @@ slots.sort((slot1, slot2) => {
     }
   }, [minDate, setDate])
 
-  return <Ui timeSlots={dayOfTimeSlots} />
+  useMemo(() => {
+    const dateIsDisabled: {[key: string]: boolean} = {}
+    var startDate = new Date(date.toLocaleDateString())
+    for(let i = 0; i < daysToFetch; i++) {
```

Comment:
I don't think this will work if a user goes forward in calendar picker and fetches a new batch of appointments